src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -151,12 +159,11 @@ enum ResolutionFailure<'a> {
         unresolved: Cow<'a, str>,
     },
     /// This happens when rustdoc can't determine the parent scope for an item.
-    ///
     /// It is always a bug in rustdoc.
```

Comment:
The part "It is always a bug in rustdoc" seems pretty important, so to include it in the summary I deleted the paragraph break.